### PR TITLE
DOC-2051: Tiny 6.5 community docs release

### PR DESCRIPTION
**Pull request by brian-forte**
DOC-2051: Tiny 6.5 community docs release

Changes:
* Added 6.5-specific tinyMCE changelog.md entries to changelog.adoc.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Comment by kemister85** [comment]
@brian-forte reminder to do a `.api-version` bump with this PR before it is merged.

File: modules/ROOT/pages/changelog.adoc [hunk 1]
```diff
@@ -4,6 +4,78 @@
...
+* New `editable_root` option that can be set to `false` to prevent editing of the editors root element.
+* New `editor.setEditableRoot` API that sets the editable state of the editor root element.
+* New `editor.hasEditableRoot` API that returns `true` or `false` depending on the editable state of the editor root element.
+* New `EditableRootStateChange` event that gets dispatched when the state of the editable root is changed.
+
+=== Improved
+* Screen readers are now able to announce the highlighted menu item of link comboboxes.
+* Now `icon` field for dialog footer `togglebutton`s is not mandatory.
+* Toolbar buttons and menu items were not disabled when they couldn't be used on noneditable content.
+* Updated toolbar "More" button tooltip text from "More..." to "Reveal or hide additional toolbar items".
```
Comment (kemister85) [inline modules/ROOT/pages/changelog.adoc:28]:
```suggestion
```

Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:28]: This has been moved to `6.6` @brian-forte

File: modules/ROOT/pages/changelog.adoc [hunk 2]
```diff
@@ -4,6 +4,78 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.5 - 2023-06-01
+
+=== Added
+* New `table_merge_content_on_paste` option which disables the merging behaviour when pasting a table inside an existing table.
+* New optional `defaultExpandedIds` and `onToggleExpand` options to the `tree` component config.
+* New optional `defaultSelectedId` option to the `tree` component config.
+* New `help_accessibility` option which displays the keyboard shortcut to access the help functionality in the statusbar.
```
Comment (kemister85) [inline modules/ROOT/pages/changelog.adoc:13]:
```suggestion
```

Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:13]: This has been moved to `6.6` @brian-forte

File: modules/ROOT/pages/changelog.adoc [hunk 3]
```diff
@@ -4,6 +4,78 @@
...
+* New `accordion` plugin with the `InsertAccordion` command.
+* New `accordion` and `accordion-toggle` icons.
```
Comment (kemister85) [inline modules/ROOT/pages/changelog.adoc:15]:
```suggestion
```

Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:15]: Suggestion to remove these as the new component has not been release. @brian-forte

File: modules/ROOT/pages/changelog.adoc [hunk 4]
```diff
@@ -4,6 +4,78 @@
...
+* Quickbar toolbars are now positioned in the middle of the selection horizontally, if the selection contains more than one table cell.
+* Exposed `dataTransfer` property of drag and drop events for elements with a `contenteditable="false"` attribute.
+
+=== Changed
+* The `caption`, `address` and `dt` elements were allowed to have non-inline children elements when the editor schema was set to `html4`.
+* SVG icons for back and foreground colors now uses class instead of id to identify SVG elements that should change color.
+* The `a` elements were allowed to have non-inline child elements when the editor schema was set to `html4`.
+
+=== Fixed
+* Right clicking on a merge tag would result in different highlighting across browsers.
```
Comment (kemister85) [inline modules/ROOT/pages/changelog.adoc:41]:
```suggestion
* Right clicking on a merge tag instance presented different highlighting depending on the host browser.
```

Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:41]: Updated to reflect the incoming changes from the approved PR

File: modules/ROOT/pages/changelog.adoc [hunk 5]
```diff
@@ -4,6 +4,78 @@
...
+* Popups were not constrained within the scrollable container when in a shadow root.
+* Pressing arrow keys inside RTL elements would move the caret in an incorrect direction when moving over elements with the `contenteditable` attribute set to `false`.
+* Inserting table consecutively without focus in the editor would result in the table being inserted at the wrong position.
+* Pasting content into the editor would not fire `beforeinput` and `input` events.
+* In some cases, the exiting a `blockquote` element could fail when the cursor was positioned at the end of the `blockquote`.
+* Templates containing an `<html>` tag were not parsed before being rendered for preview.
+* Typing after deleting formatted content could remove a space at the start of the typing.
+* Invalid markup in Notification and Dialog close buttons.
+* In dialogs, an incorrect `aria-describedby` attribute caused the body of the dialog to be announced when using a screen reader.
+* The sticky toolbar would not be rendered correctly when transitioning from the custom editor view to the main view.
```
Comment (kemister85) [inline modules/ROOT/pages/changelog.adoc:77]:
```suggestion
* Text within anchor tags, <a>, presented with the Times New Roman font-family, ignoring the font family used in the original document.
* The sticky toolbar would not be rendered correctly when transitioning from the custom editor view to the main view.
```

Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:77]: Missing changelog entry from TINY-9812

**Review by kemister85 — APPROVED** [review]

File: modules/ROOT/pages/changelog.adoc (hunk 4 above)
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc]: To quote my commit message:
> Incorporated copy-edit provided by @kemister85 in the currently open PR.

File: modules/ROOT/pages/changelog.adoc (hunk 1 above)
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc:28]: The suggested change and the original are identical. Consequently, I’m resolving this one.

**Review by MitchCTiny — APPROVED** [review]

File: modules/ROOT/pages/changelog.adoc (hunk 2 above)
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc:13]: To quote my commit message:

> Removed an entry from Improved section of 6.5-specific changes:
> 
> * New `help_accessibility` option which displays the keyboard shortcut to access the help functionality in the statusbar.
> 
> This change has been moved to the 6.6 release.

File: modules/ROOT/pages/changelog.adoc (hunk 5 above)
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc]: This is a change to the PowerPaste Premium plugin and is set to be recorded as such in the PowerPaste changelog, as per the now-merged [PR176](https://github.com/tinymce/powerpaste2/pull/176/files).

File: modules/ROOT/pages/changelog.adoc [hunk 6]
```diff
@@ -4,6 +4,78 @@
...
+=== Improved
+* Screen readers can now announce highlighted items listed in the Link dialog’s link combobox.
+* The `icon` field for a dialog’s footer `togglebutton` is no longer mandatory.
+* Toolbar buttons and menu items now present as disabled when they cannot be used because a selected element has a `contenteditable="false"` attribute.
+* Help text displayed at *Help -> Help -> Keyboard Navigation* re-written.
+* Translations added for Help text displayed at *Help > Help > Keyboard Navigation*.
+* For word count purposes these characters are now considered punctuation marks: *$*, *~*, *+*, *|*, *№*, and *`*. They no longer increase a document’s word count.
+* Updated the `codesample` plugin dialog and the `template` plugin dialog to use the `listbox` component to match other dialogs.
+* If the selection contains more than one table cell, Quickbar toolbars are now positioned in the middle of the selection horizontally.
+* The `dataTransfer` property of drag-and-drop events for elements with a `contenteditable="false"` attribute is now exposed.
```
Comment (kemister85) [inline modules/ROOT/pages/changelog.adoc:32]:
```suggestion
* Exposed `dataTransfer` property of drag and drop events for elements with a `contenteditable="false"` attribute.
```

Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:34]: Update based on incoming change from https://github.com/tinymce/tinymce-docs/pull/2750#discussion_r1211059877 @mkzhx

**Review by kemister85 — COMMENTED** [review]
Incoming changes from https://github.com/tinymce/tinymce-docs/pull/2750#discussion_r1211059877

File: modules/ROOT/pages/changelog.adoc [hunk 7]
```diff
@@ -4,6 +4,80 @@
...
+* The Table toolbar was visible even if the table was within a host element with a `contenteditable="false"` attribute set.
+* Quickbar toolbars were incorrectly shown for elements with a `contenteditable="false"` attribute set in a root with a `contenteditable="false"` attribute set.
+* When Chrome was the host browser, adding a newline when the insertion point was placed after a table could, in some specific situations, not generate the expected newline.
+* Show the calculated height and width of Media Embed elements in the `media` plugin dialog.
+* Removing an image that failed to upload from an empty paragraph left the paragraph without a padding `<br>` tag.
+* Allow a Media Embed element to be correctly resized when using the `media` plugin dialog by converting the Media Embed element to a standalone iframe.
+* In some circumstances, an inline alert in the _Search and Replace_ dialog persisted when it was not necessary.
+* Context toolbars displayed the incorrect status for the `advlist` plugin buttons.
+* In Safari running on iOS, Korean characters merged onto the previous line upon typing after inserting a newline by pressing Enter.
```
Comment (kemister85) [inline modules/ROOT/pages/changelog.adoc:57]:
```suggestion
* On iOS Safari, Korean characters will no longer merge onto the previous line after inserting a newline by pressing Enter.
```

Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:57]: Incoming change from https://github.com/tinymce/tinymce-docs/pull/2748#discussion_r1211041664 @mkzhx

**Review by kemister85 — COMMENTED** [review]
Incoming change from https://github.com/tinymce/tinymce-docs/pull/2748#discussion_r1211041664

File: modules/ROOT/pages/changelog.adoc (hunk 7 above)
Comment (mkzhx) [reply modules/ROOT/pages/changelog.adoc:57]: This won't be necessary as it was adapted from the latest version of my changelog entry already.

File: modules/ROOT/pages/changelog.adoc [hunk 8]
```diff
@@ -4,6 +4,81 @@
...
+* The content of the dialog body could not be scrolled.
+* Some toolbar items, when in a not-enabled state, did not render the `not-allowed` mouse pointer.
+* Formats were incorrectly applied to the closest editable element if the selection was in a `contenteditable="false"` context.
+* Formats were incorrectly removed from the closest editable element if the selection was in a `contenteditable="false"` context.
+* Formatter API, `canApply`, was not returning `false` when the selection was in a `contenteditable="false"` context.
+* When dragging image elements and dropping the image in the editor the `dragend` event would sometimes not fire when Firefox was the host browser.
+* It was possible to remove links in noneditable contents with the `unlink` editor command.
+* Direction was not visually changing when using the Directionality plugin on an element which had the `direction` CSS property set.
+* Whitespace between transparent elements was incorrectly converted into empty paragraphs.
+* Popups were not constrained within the scrollable container when in a shadow root.
```
Comment (kemister85) [inline modules/ROOT/pages/changelog.adoc:70]:
```suggestion
* Context menus and menu items were not constrained within the scrollable container if the parent is a Shadow Host
```

Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:70]: @brian-forte Incoming changes from https://github.com/tinymce/tinymce-docs/pull/2771/files 
@shanmen-tiny  FYI

**Review by kemister85 — COMMENTED** [review]
Incoming changes from https://github.com/tinymce/tinymce-docs/pull/2771/files

File: modules/ROOT/pages/changelog.adoc (hunk 3 above)
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc:15]: The according plugin has made it into 6.5 Community.

File: modules/ROOT/pages/changelog.adoc (hunk 7 above)
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc:58]: Edits made upstream.

File: modules/ROOT/pages/changelog.adoc (hunk 8 above)
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc:71]: Edits made upstream.

File: modules/ROOT/pages/changelog.adoc [hunk 9]
```diff
@@ -4,6 +4,86 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.5 - 2023-06-05
```
Comment (kemister85) [inline modules/ROOT/pages/changelog.adoc:7]: Reminder to update this to match the release-date.

Comment (brian-forte) [inline modules/ROOT/pages/changelog.adoc:7]:
```suggestion
== 6.5 - 2023-06-06
```

Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc]: Done.

**Pull request merged**